wagtail/contrib/modeladmin/templates/modeladmin/choose_parent.html

```diff
@@ -20,12 +20,20 @@
 {% block content %}
 
     {% block header %}
-        {% include "modeladmin/includes/header_with_breadcrumb.html" with title=view.get_page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
+        {% include "wagtailadmin/shared/header_with_locale_selector.html" with title=view.get_page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
     {% endblock %}
 
     <div>
 
         <div class="nice-padding">
+            <p class="back">
+                <a href="{{ view.index_url }}">
+                    <svg class="icon icon-arrow-left arrow_left_icon default" aria-hidden="true">
```

Comment:
```suggestion
                    <svg class="icon default" aria-hidden="true">
```

`icon-arrow-left arrow_left_icon ` is not needed - this was the old approach, the `svg` element is now the icon.